pipeline_dp/pipeline_operations.py

```diff
@@ -1,7 +1,6 @@
 """Adapters for working with pipeline frameworks."""
 
 import random
-import collections
```

Comment:
Probably it's an incorrect merge in git. Please return back

Reply:
Done.